templates/repo/settings/webhook/base_list.tmpl

```diff
@@ -62,7 +62,7 @@
 				<span class="text {{if eq .LastStatus 1}}green{{else if eq .LastStatus 2}}red{{else}}grey{{end}} gt-mr-3">{{svg "octicon-dot-fill" 22}}</span>
 				<a class="text truncate gt-f1 gt-mr-3" title="{{.URL}}" href="{{$.BaseLink}}/{{.ID}}">{{.URL}}</a>
 				<a class="muted gt-p-3" href="{{$.BaseLink}}/{{.ID}}">{{svg "octicon-pencil"}}</a>
-				<a class="delete-button gt-p-3" data-url="{{$.Link}}/delete" data-id="{{.ID}}">{{svg "octicon-trash"}}</a>
+				<a class="delete-button gt-p-3" data-url="{{$.BaseLink}}/delete" data-id="{{.ID}}">{{svg "octicon-trash"}}</a>
```

Comment:
This is because `POST /admin/hooks/delete` was moved to `POST /admin/hooks/{configType:system|default}/delete`